SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/zsetasqlTests.java

```diff
@@ -62,8 +63,8 @@ public void simpleParseTest() {
                 "ARRAY<STRUCT<DATE, DATE, DATE, DATE, DATE>>[\n" +
                 "  {0001-01-01, 1969-01-01, 1970-01-01, 2014-01-01, 9999-12-31}\n" +
                 "]";
-        Zetatest.zetalexer lexer = new Zetatest.zetalexer(CharStreams.fromString(test));
-        Zetatest.zetatest parser = new Zetatest.zetatest(new CommonTokenStream(lexer));
+        zetalexer lexer = new zetalexer(CharStreams.fromString(test));
```

Comment:
Rename zetatest.g4 and zetalexer.g4 to Zetatest.g4 and Zetalexer.g4, and capitalize those names inside those files. Then the class names will also be capitalized correctly here.